Formula/snap.rb

```diff
@@ -0,0 +1,56 @@
+class Snap < Formula
+  desc "The snap tool enable systems to work with .snap files"
+  homepage "https://snapcraft.io/"
+  license "GPL-3.0-only"
+
+  stable do
+    version "2.46"
+    url "https://github.com/snapcore/snapd/releases/download/#{version}/snapd_#{version}.vendor.tar.xz"
+    sha256 "c4f532018ca9d2a5f87a95909b3674f8e299e97ba5cb5575895bcdd29be23db3"
+  end
+
+  head do
+    url "https://github.com/snapcore/snapd.git"
+  end
+
+  depends_on "go" => :build
+  depends_on "squashfs"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOCACHE"] = "#{HOMEBREW_CACHE}/go_cache"
```

Comment:
What is this for?

Reply:
This is for faster rebuilds, I guess it makes sense locally, can be removed if desired

Reply:
We already set it in `brew`. Yea, remove it please.